api/src/main/resources/workbench-api.yaml

```diff
@@ -1807,25 +1835,24 @@ paths:
           schema:
             type: string
       requestBody:
-        description: The admin request to publish workspace
+        description: The admin request to mark workspace as feature
         content:
           application/json:
             schema:
-              $ref: '#/components/schemas/PublishWorkspaceRequest'
+              $ref: '#/components/schemas/MarkWorkspaceFeaturedRequest'
       responses:
         200:
-          description: Workspace has been published
+          description: Workspace has been marked as featured
           content:
             application/json:
               schema:
                 $ref: '#/components/schemas/EmptyResponse'
-  /v1/admin/workspaces/{workspaceNamespace}/unpublish:
+  /v1/admin/workspaces/{workspaceNamespace}/unmarkAsFeaturedWorkspace:
```

Comment:
need to update the path in L1823 from `/publish` to match this.

Alternative idea: what do you think about using the same path for these endpoints?  One can be POST and the other can be DELETE.